src/test/java/com/feedzai/commons/sql/abstraction/engine/impl/abs/EngineGeneralTest.java

```diff
@@ -2138,6 +2141,37 @@ public void testUnionAll() throws DatabaseEngineException {
         assertEquals("COL5 must be d", "d", resultSorted.get(4));
     }
 
+    @Test
+    public void testValues() throws DatabaseEngineException {
+        final Values values = values("id", "name")
```

Comment:
I was assuming a different syntax for this:

```
values("id", "name")
    .row(k(1), k("ana"))
    .row(k(2), k("fred"))
    .row(k(3), k("manuel"))
    .row(k(4), k("rita"));
```

Does it make sense?  
Or is it trickier to use?

Reply:
It makes sense and makes our interface simpler. Going to change it